priv/Oppgave_2.livemd

```diff
@@ -0,0 +1,133 @@
+# Kontroller LED-lys med knapper
+
+## Introduksjon
+
+Nå som vi er blitt litt kjent med LED-lysene og kan styre lystilstanden programmatisk med Nerves, er neste oppgave å heller styre lystilstanden ved knappetrykk.
+
+## Fest en pHat-modul til Raspberry Pi-boardet
+
+For å kunne styre LED-lysene ved hjelp av knapper må vi først ha en knapp å trykke på. Finn derfor en `Qwiic pHat`-modul som du kan feste på I/O-pinsene til boardet – du vet, den lange, svarte raden med pinner som stikker opp. Husk å feste `Qwiic`-modulen slik at modulen peker innover kretskortet og ikke utover.
```

Comment:
Kanskje vi kan fjerne denne da de må ha skjermen for å kunne se ipadressen uansett?